travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.3"
+  - "3.4"
+
+sudo: false
+
+install: true
+
```

Comment:
Ok, so the travis build is failing because the name of file needs to be `.travis.yml`. Notice the leading '.'.

Use `git mv travis.yml .travis.yml` to rename the file. Then create a new commit and push it. That should clear up the error.